contrib/guix/libexec/build.sh

```diff
@@ -69,13 +69,22 @@ unset CPLUS_INCLUDE_PATH
 unset OBJC_INCLUDE_PATH
 unset OBJCPLUS_INCLUDE_PATH
 
-export C_INCLUDE_PATH="${NATIVE_GCC}/include"
-export CPLUS_INCLUDE_PATH="${NATIVE_GCC}/include/c++:${NATIVE_GCC}/include"
-export OBJC_INCLUDE_PATH="${NATIVE_GCC}/include"
-export OBJCPLUS_INCLUDE_PATH="${NATIVE_GCC}/include/c++:${NATIVE_GCC}/include"
+case "$HOST" in
+    *darwin*)
+        # Required for qt/qmake
+        export C_INCLUDE_PATH="${NATIVE_GCC}/include"
+        export CPLUS_INCLUDE_PATH="${NATIVE_GCC}/include/c++:${NATIVE_GCC}/include"
+        ;;
+    *)
+        export C_INCLUDE_PATH="${NATIVE_GCC}/include"
+        export CPLUS_INCLUDE_PATH="${NATIVE_GCC}/include/c++:${NATIVE_GCC}/include"
+        export OBJC_INCLUDE_PATH="${NATIVE_GCC}/include"
```

Comment:
If I am reading this right, this sets OBJC vars for non-darwin hosts. Is that intended? I would have expected the opposite.

Reply:
Heh, yea. These should/would only be needed for macOS, and only for Qt-related code. I guess everything still currently builds even with them unset, because when we compile our objc++ code, we would never be looking in any GCC related include/std-lib/related dirs in any case? Will push some changes.